pkg/k8splan/watcher.go

```diff
@@ -153,64 +192,111 @@ func (w *watcher) start(ctx context.Context) {
 				}
 			}
 
+			// Check to see if we've exceeded our failure count threshold
+			var maxFailureThreshold int
+			if rawMaxFailureThreshold, ok := secret.Data[maxFailuresKey]; ok {
+				// max failure threshold is defined. parse and compare
+				maxFailureThreshold, err = strconv.Atoi(string(rawMaxFailureThreshold))
+				if err != nil {
+					maxFailureThreshold = int(^uint(0) >> 1) // max int
```

Comment:
If no threshold is defined, is our default threshold 9,223,372,036,854,775,807?

Reply:
Yes. I guess I could also just set it to `-1` and have logic to handle that case below, but by setting it to max int should provide more than ample amounts of attempts of a plan in my opinion.